presto-tpch/src/main/java/com/facebook/presto/tpch/TpchMetadata.java

```diff
@@ -408,7 +408,9 @@ public ColumnMetadata getColumnMetadata(ConnectorSession session, ConnectorTable
 
     private Predicate<NullableValue> convertToPredicate(TupleDomain<ColumnHandle> predicate, TpchColumn column)
     {
-        return nullableValue -> predicate.contains(TupleDomain.fromFixedValues(ImmutableMap.of(toColumnHandle(column), nullableValue)));
```

Comment:
can you please provide a test?

Reply:
This is covered by `testStatsCalculatorUsesLayout` on CBO... Can you help me with this?